### ISSUE TITLE
When training Deeplabv2, did you use any pre-trained model? 

### ISSUE THREAD

**Issue by CenIII** [jiwoon-ahn/irn#21]
When you were training Deeplabv2 using the pseudo-labels produced by your method, had the Deeplabv2 been pretrained before hand or just raw? Thanks.

**Comment by jiwoon-ahn** [comment]
Hi @CenIII,
We used imagenet pretrained model for deeplabv2.

**Comment by CenIII** [comment]
> Hi @CenIII,
> We used imagenet pretrained model for deeplabv2.

The resnet backbone of deeplabv2 seems to have different configurations (e.g. different dilation rate or stride). How did you reload the weights of the pretrained model? (My understanding is that the difference in dilation rate wont affect the weight loading?)

Second question, did you freeze the first bn layer when training? 

Thanks!

**Comment by jiwoon-ahn** [comment]
1. Your understanding is right. We just followed the way of the original deeplab.
2. Yes, the first conv and bn should be freezed when finetuning.

**Comment by CenIII** [comment]
> 1. Your understanding is right. We just followed the way of the original deeplab.
> 2. Yes, the first conv and bn should be freezed when finetuning.

Thank you for replying! 

Would you like to share the open source code you used to train deeplabv2?  Thank you!

**Comment by Ien001** [comment]
The same question as @CenIII , Thank you! @jiwoon-ahn

**Comment by jiwoon-ahn** [comment]
@CenIII,
We made our own model from the bottom. I highly recommend to use the code of the author of DeepLab (https://bitbucket.org/aquariusjay/deeplab-public-ver2/src/master/)

**Comment by CenIII** [comment]
@jiwoon-ahn Thanks! What do you mean by "made from bottom"? So what's the difference of your model and training process from the original one?

**Comment by jiwoon-ahn** [comment]
I believe none or little gap exists between our model from the original as we did our best to minimize the difference.

**Comment by CenIII** [comment]
> I believe none or little gap exists between our model from the original as we did our best to minimize the difference.

Got it. Thank you for your help!
